lib/Access.php

```diff
@@ -1486,11 +1485,7 @@ public function areCredentialsValid($name, $password) {
 		if (!$testConnection->setConfiguration($credentials)) {
 			return false;
 		}
-		try {
```

Comment:
this fail a bunch  of acceptance tests I guess. - WebDav requests are expect 4xx status instead of 5xx